frontend/lib/src/components/transaction_component/transaction_component.dart

```diff
@@ -69,21 +69,10 @@ class TransactionComponent extends StatelessWidget {
       ]
     };
 
-    // Manually set category for now as there is no plan for this feature as of now
-    // TODO: Transfer type texts
-    final Map<TransactionTypes, Map<String, dynamic>> typeTexts = {
-      TransactionTypes.CREDIT: {
-        'category': TransactionCategories.BILLS,
-        'success': 'withdrawn'
-      },
-      TransactionTypes.DEPT: {
-        'category': TransactionCategories.SAVINGS,
-        'success': 'deposited',
-      },
-      TransactionTypes.TRANSFER: {
-        'category': TransactionCategories.SENDER,
-        'success': 'transfered'
-      },
+    final Map<TransactionTypes, String> typeTexts = {
```

Comment:
```suggestion
    final Map<TransactionTypes, String> x = {
```
rename to a more descriptive variable